pkg/band/channel_mask.go

```diff
@@ -446,8 +446,8 @@ func generateChMask72Generic(currentChs, desiredChs []bool, atomic bool) ([]ChMa
 		// We also sort the masks descending on the number of enabled channels in order to ensure
 		// that intermediary states through which the end device goes while parsing the masks
 		// are valid (i.e. the total number of enabled channels is always greater than 0).
-		slices.SortFunc(pairs, func(a, b ChMaskCntlPair) bool {
-			return trueCount(a.Mask[:]...) >= trueCount(b.Mask[:]...)
+		slices.SortFunc(pairs, func(a, b ChMaskCntlPair) int {
```

Comment:
Go 1.21 will bring the `slices` and `maps` packages to the standard library, and the variant adopted there uses an `int` returning comparator instead of a `bool` one. The change here allows us to easily move to the standard library implementation once we move to Go 1.21.